safe_autonomy_simulation/sims/spacecraft/point_model.py

```diff
@@ -189,6 +191,7 @@ def __init__(
         state_dot_min: typing.Union[float, np.ndarray] = -np.inf,
         state_dot_max: typing.Union[float, np.ndarray] = np.inf,
         integration_method: str = "RK45",
+        use_jax: bool = False
```

Comment:
The dynamics can specify whether jax should be used, but the models using them don't have a way to pass that down.  The dynamics will always be forced to not use jax.

Reply:
I've exposed jax to the `safe_autonomy_simulation.sims` entities and the point mass integrators